<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierReferenceClass</class>
 <widget class="QDialog" name="modifierReferenceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modifierReference</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_identifiantModifier">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrez le code  ISSN ou ISBN de la référence à modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_auteursModifier">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saisissez le(s) nouveau(x) nom(s) de(s) l'auteur(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modifier">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
